frontend/src/metabase/actions/containers/ActionCreator/ActionCreator.unit.spec.tsx

```diff
@@ -90,6 +122,27 @@ describe("ActionCreator", () => {
       expect(screen.getByRole("button", { name: "Save" })).toBeDisabled();
       expect(screen.getByRole("button", { name: "Cancel" })).toBeEnabled();
     });
+
+    it("should show data reference icon and is clickable", async () => {
```

Comment:
"is clickable" here sounds as if it's related to `it` rather than the data reference icon. Should it be something like "should show clickable data reference icon"?